Add Docker Desktop 4.7.0 installer command line docs (#14455)
docker/docker.github.io: desktop/mac/install.md

    For more information, see [Docker Desktop License Agreement](../../subscription/index.md#docker-desktop-license-agreement). We recommend that you also read the [Blog](https://www.docker.com/blog/updating-product-subscriptions/){: target="_blank" rel="noopener" class="_" id="dkr_docs_desktop_install_btl"} and [FAQs](https://www.docker.com/pricing/faq){: target="_blank" rel="noopener" class="_" id="dkr_docs_desktop_install_btl"} to learn how companies using Docker Desktop may be affected.

### Install from the command line

After downloading `Docker.dmg`, to install Docker Desktop to the Applications folder, run for instance the following commands in a terminal:
```
sudo hdiutil attach Docker.dmg

> sudo /Volumes/Docker/Docker.app/Contents/MacOS/install --accept-license

Reviewer: I'm not sure we want to put the --accept-license in the example here. I understand why it's there (to show where the arguments go) but I don't want people copy- and pasting it and then saying that they never knowingly accepted the license.

Author: Good point, I’ll remove the arguments from the example.

sudo hdiutil detach /Volumes/Docker
```

The installer accepts the following flag:

> - `--accept-license`: disables prompting for license acceptance on Docker Desktop first run

Reviewer: Let's rephrase this to imply "you are now accepting the license" rather than "this disables having to accept the license".

`--accept-license`: accepts the license now, rather than requiring it to be accepted when the application is first run

Is there a copy of the license online that we can link to?

Author: > Is there a copy of the license online that we can link to?

Yes, there is https://www.docker.com/legal/docker-subscription-service-agreement/

    For more information, see [Docker Desktop License Agreement](../../subscription/index.md#docker-desktop-license-agreement). We recommend that you also read the [Blog](https://www.docker.com/blog/updating-product-subscriptions/){: target="_blank" rel="noopener" class="_" id="dkr_docs_desktop_install_btl"} and [FAQs](https://www.docker.com/pricing/faq){: target="_blank" rel="noopener" class="_" id="dkr_docs_desktop_install_btl"} to learn how companies using Docker Desktop may be affected.

### Install from the command line

> After downloading `Docker.dmg`, to install Docker Desktop to the Applications folder, run for instance the following commands in a terminal:

Reviewer 2: ```suggestion
After downloading `Docker.dmg`, run the following commands in a terminal to install Docker Desktop in the Applications folder :
```

> ```

Reviewer 2: nit: add a blank line before the code

> ```

Reviewer 2: ```suggestion
```console
```

sudo hdiutil attach Docker.dmg
sudo /Volumes/Docker/Docker.app/Contents/MacOS/install
sudo hdiutil detach /Volumes/Docker
```

> The `install` command accepts the following flag:

Reviewer 2: ```suggestion
The `install` command accepts the following flags:
```

> The `install` command accepts the following flag:

Reviewer: flags

> - `--accept-license`: accepts the [Docker Subscription Service Agreement](https://www.docker.com/legal/docker-subscription-service-agreement) now, rather than requiring it to be accepted when the application is first run

Reviewer 2: ```suggestion
- `--accept-license`: accepts the [Docker Subscription Service Agreement](https://www.docker.com/legal/docker-subscription-service-agreement){: target="_blank" rel="noopener" class="_"} during installation, instead of prompting you to accept the license when the application is first run
```

Author: Here as well, the user running the application can be a different user than the «you» installing it.

> - `--allowed-org=<org name>`: requires the user to sign in and be part of the specified Docker Hub organization when running the application

Reviewer 2: ```suggestion
- `--allowed-org=<org name>`: requires you to sign in and be part of the specified Docker Hub organization when running the application
```

Author: Well it’s very likely the user installing the software from the command line will be different than the user running it.
The command line installation is mainly targeted at company admins.

Reviewer 2: Sure, makes sense.
